conference.js

```diff
@@ -1583,6 +1584,7 @@ export default {
             APP.UI.mucJoined();
             APP.API.notifyConferenceJoined(APP.conference.roomName);
             APP.UI.markVideoInterrupted(false);
+            APP.store.dispatch(setInfoDialogVisibility(true));
```

Comment:
Wouldn't it be better if we add this in a middleware in the info dialog plugin on CONFERENCE_JOINED? We already have several other features listening for the same event in there middleware, so it looks like the right way to go if we want to keep the feature independent from other code.

Reply:
Moved to middleware. I squashed some changes and force pushed to preserve the separation of the commits.